src/main/client/src/common/modals/DeleteBookModal.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { Button, Header, Icon, Modal } from 'semantic-ui-react';
+
+export default class DeleteBookModal extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            open: false
+        };
+        this.close = this.close.bind(this);
+    }
+
+    close = () => {
+        this.setState({ open: false })
+    }
+
+    render() {
+        return (
+            <div>
+                <Modal trigger={
+                    <button id="delBookBtn" onClick={() => this.setState({open: true})}>
+                        Delete Selected Book(s)
+                    </button>
+                } basic size='small' open={this.state.open}>
+                    <Header id="modalHeader" icon='trash alternate' content='Delete Selected Books' />
+                    <Modal.Content>
+                        <p>
+                            You are about to permanently delete the selected books. Do you wish to continue?
+                        </p>
+                    </Modal.Content>
+                    <Modal.Actions>
+                        <Button id="noBtn" basic color='red' inverted onClick={this.close}>
+                            <Icon name='remove' />No
```

Comment:
Should button colours be switch because deleting is a negative action (genuine question)

Also, Semantic allows you do to state whether a button is positive or negative (can't remember the wording), might be a bit more descriptive

Reply:
changed to "Cancel" and "Delete" where delete is now red and cancel is white

Reply:
Sounds good